config/default.json

```diff
@@ -9,6 +9,11 @@
     "level": "info"
   },
   "server": {
-    "port": "8080"
+    "port": "8080",
+    "request": {
+      "payload": {
+        "limit": "10mb"
```

Comment:
do we also want to expose this as an environment variable?

Reply:
done